apollo/client.ts

```diff
@@ -6,8 +6,9 @@ import {
 } from "@apollo/client";
 import { request } from "http";
 
+// this has the latest subgraph for now, may need to be updated later
 const umaLinkKovan = new HttpLink({
-  uri: "https://api.thegraph.com/subgraphs/name/umaprotocol/uma-kovan",
+  uri: "https://api.thegraph.com/subgraphs/name/nicholaspai/kovan-staging",
```

Comment:
I've re-indexed this subgraph at `nicholaspai/kovan-contracts-staging` and will deprecate `kovan-staging` going forwards

Reply:
Just out of curiosity, why isn't this under an UMA namespace?